Hdrp/add documentation link

RSlysz opened the pull request:
### Purpose of this PR
Add documentation links in component's Inspector header bar (HDRP + some Core preparation)
Also added unit test on links (see below).

*Known issue:*
- This require the documentation to make every documentation page live in the same namespace than the actual component (so moving the Volumes in Core with inside the webpage links for HDRP and URP etc.)
- The links of built-in component Camera, Light and ReflectionProbe that we only overide the inspector cannot be updated. The acurate link will be on the corresponding AdditionalData component for now.

*Workflow change for release:*
- When releasing, if documentation version need to be updated (it need to be done for all page at target version), change the Documention.version. Documentation is a file at base of Runtime folder of each package (HDRP and CoreRP for now)

---
### Release Notes
Fixed documentation links in component's Inspector header bar.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2Fadd-documentation-link&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

**Manual Tests**: What did you do?
- [x] Opened test project + Run unit tests
- [ ] Built a player
- [x] C# and shader warnings (supress shader cache to see them)

**Automated Tests**: 
- Added Unit test grabbing all link and testing if they are accessible for CoreRP (but disabled it for now as documentation link are not done yet in CoreRP)
- Added Unit test grabbing all link and testing if they are accessible for HDRP

---
### Overall Product Risks
**Technical Risk**: Low
**Halo Effect**: Low

---
### Comments to reviewers
@iTris666 for notification

*QA:*
- The documentation link should go to 6.9 documentation link (the latest released documentation at the moment) See also *known issue* above.

TomasKiniulis commented:
Diffusion Profile is currently missing a link
https://confluence.hq.unity3d.com/display/HDRP/Component+documentation+links

sebastienlagarde commented on com.unity.render-pipelines.high-definition/Runtime/Documentation.cs:
```diff
@@ -0,0 +1,21 @@
+
+        //Update this field when upgrading the target Documentation for the package
+        //Should be linked to the package version somehow.
+        internal const string version = "6.9";
```
let's use same version for all packages, otherwise update will be a problem. Also maybe try to centralise the version in a version.cs file instead. (like we have a hlsl version.hlsl)

RSlysz replied:
Fixed in 42f74e1

RSlysz commented:
@TomasKiniulis I fixed the missing DiffusionProfile link.
Also, I moved Volume, VolumeProfile and EnvironmentLibrary link to HDRP documentation for the moment. (It fix missing gocumentation for volume part and EnvironmentLibrary documentation should land soon)
